lib/faker/aws/en.ex

```diff
@@ -0,0 +1,207 @@
+defmodule Faker.Aws.En do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Functions for generating Aws information in English
+  """
+
+  @doc """
+  Returns a random region from the available Aws regions
+
+  ## Examples
+
+      iex> Faker.Aws.En.region()
+      "Asia Pacific (Tokyo)"
+      iex> Faker.Aws.En.region()
+      "US East (Ohio)"
+      iex> Faker.Aws.En.region()
+      "Europe (Milan)"
+      iex> Faker.Aws.En.region()
+      "Africa (Cape Town)"
+  """
+
+  @spec region() :: String.t()
+  sampler(:region, [
+    "US East (N. Virginia)",
+    "US East (Ohio)",
+    "US West (N. California)",
+    "US West (Oregon)",
+    "Africa (Cape Town)",
+    "Asia Pacific (Hong Kong)",
+    "Asia Pacific (Mumbai)",
+    "Asia Pacific (Seoul)",
+    "Asia Pacific (Singapore)",
+    "Asia Pacific (Sydney)",
+    "Asia Pacific (Tokyo)",
+    "Canada (Central)",
+    "Europe (Frankfurt)",
+    "Europe (Ireland)",
+    "Europe (London)",
+    "Europe (Milan)",
+    "Europe (Paris)",
+    "Europe (Stockholm)",
+    "Middle East (Bahrain)",
+    "South America (São Paulo)"
+  ])
```

Comment:
I'm curious why you decided to use the long region names instead of region codes here.

Reply:
Why I ask is because I typically default to region codes when thinking of AWS regions rather than the name.

Reply:
It is biased toward my own use, I use faker to generate random data that is human-readable. I could introduce a function in another PR to provide the user with the choice between calling Faker.Aws.En.region/0 and Faker.Aws.En.region_code/0. The second function would generate random data using the region codes. It is an implementation detail but I could also transform Faker.Aws.En.region/0 to Faker.Aws.En.Region/1 and if the user wants the region code they could ask it by passing an argument.

Reply:
Thanks for the context. Would you be willing to rename this function to `region_name` and introduce a `region_code` function in either this or a subsequent pull request? I know I'd get tripped up on this.

Reply:
100%, I will PR something later today after 5 PM (GMT-4)

Reply:
Awesome! Thank you!